torch/multiprocessing/reductions.py

```diff
@@ -21,43 +21,6 @@
     pass
 
 
-class StorageWeakRef:
```

Comment:
It is NOT ok to remove public APIs without appropriate care.
In this case, I don't think it is worth breaking existing user code by removing this altogether.
You can import StorageWeakRef from torch.utils.weak here to preserve the current public API.

Reply:
Looks like some tests are still failing.